addons/skin.estuary/1080i/DialogAddonInfo.xml

```diff
@@ -8,45 +8,101 @@
 		<control type="group">
 			<include>OpenClose_Right</include>
 			<control type="button">
-				<left>770</left>
-				<top>162</top>
-				<width>1040</width>
-				<height>559</height>
+				<left>605</left>
+				<top>400</top>
+				<width>1235</width>
+				<height>375</height>
 				<label>-</label>
 			</control>
 			<control type="textbox">
-				<left>840</left>
-				<top>200</top>
-				<width>900</width>
-				<height>380</height>
+				<left>670</left>
+				<top>435</top>
+				<width>1050</width>
+				<height>300</height>
 				<font>font_textbox</font>
 				<label>$INFO[ListItem.AddonSummary,[B],[/B][CR]][COLOR=white]$INFO[ListItem.AddonDescription][/COLOR][COLOR=white][CR][/COLOR]</label>
 				<textcolor>button_focus</textcolor>
 			</control>
-			<control type="textbox">
-				<left>840</left>
-				<top>610</top>
-				<width>900</width>
-				<height>70</height>
-				<font>font_textbox</font>
-				<label fallback="31008">$INFO[ListItem.AddonDisclaimer,$LOCALIZE[24052]: ]</label>
-				<textcolor>button_focus</textcolor>
+			<control type="group">
+				<left>625</left>
+				<top>176</top>
+				<visible>Integer.IsEqual(Container(50).NumItems,0)</visible>
+				<control type="image">
+					<width>385</width>
+					<height>220</height>
+					<texture border="2" background="true">DefaultNoPreview.png</texture>
+					<aspectratio>scale</aspectratio>
+					<bordertexture colordiffuse="border_alpha">colors/black.png</bordertexture>
+					<bordersize>4</bordersize>
+				</control>
+				<control type="image">
+					<left>405</left>
+					<width>385</width>
+					<height>220</height>
+					<texture border="2" background="true">DefaultNoPreview.png</texture>
+					<aspectratio>scale</aspectratio>
+					<bordertexture colordiffuse="border_alpha">colors/black.png</bordertexture>
+					<bordersize>4</bordersize>
+				</control>
+				<control type="image">
+					<left>810</left>
+					<width>385</width>
+					<height>220</height>
+					<texture border="2" background="true">DefaultNoPreview.png</texture>
+					<aspectratio>scale</aspectratio>
+					<bordertexture colordiffuse="border_alpha">colors/black.png</bordertexture>
+					<bordersize>4</bordersize>
+				</control>
 			</control>
-			<control type="image">
-				<left>790</left>
-				<top>600</top>
-				<width>1000</width>
-				<height>3</height>
-				<texture colordiffuse="button_focus" border="2">dialogs/separator-grey.png</texture>
+			<control type="panel" id="50">
+				<left>625</left>
+				<top>176</top>
+				<width>1215</width>
+				<height>233</height>
+				<ondown>9000</ondown>
+				<onup>9000</onup>
+				<orientation>horizontal</orientation>
+				<itemlayout height="225" width="405">
+					<control type="image">
+						<width>385</width>
+						<height>220</height>
+						<texture border="2" fallback="DefaultAddonNone.png" background="true">$INFO[ListItem.Icon]</texture>
```

Comment:
Yeah should be DefaultNoPreview.png. But that's not the issue. If the screenshots are there than the  icon is not empty (has an url) and the fallback will not kick in and you just have to wait for the image to be downloaded. The fallback can be removed as it's not used here. What I can do is show DefaultNoPreview.png always behind the screenshots so that it shows that in case of slow connection.